spec/services/feeds/import_spec.rb

```diff
@@ -63,28 +63,54 @@
       end
     end
 
-    # it "reports an article creation error" do
-    #   allow(described_classs).to receive(:create_articles_from_user_feed).and_raise(StandardError)
-    #   allow(Honeybadger).to receive(:notify)
+    it "queues as many slack messages as there are articles", vcr: { cassette_name: "feeds_import" } do
```

Comment:
We do have `configure_rspec_metadata!` enabled, so it would infer the cassette name from the spec description:

```suggestion
    it "queues as many slack messages as there are articles", :vcr do
```

Reply:
I want to avoid that, I'd prefer to recycle the same spec instead of having multiple VCR recordings with the same data